### PR TITLE
Chapter 11 Blade

### PR DESCRIPTION
**Pull request by dvson-sun**
View Users:
image
View Orders:
image

### CODE REVIEW

**Comment by dvson-sun** [comment]
ready

File: resources/views/layouts/header.blade.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <li class="nav-item" style="margin-left: 30px;">
+            <a class="dropdown-item" href="{{ route('logout') }}" onclick="event.preventDefault();document.getElementById('logout-form').submit();">
+                <button class="btn btn-primary">{{ __('Log Out') }} </button>
+            </a>
+            <form id="logout-form" action="{{ route('logout') }}" method="POST" class="d-none">
+                @csrf
+            </form>
+        </li>
+    </ul>
+</nav>
```
Comment (huyenttm-0901) [inline resources/views/layouts/header.blade.php:26]: end of file

File: resources/views/layouts/header.blade.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <a class="navbar-brand" style="margin-left: 50px;" href="#"><span>Adminitrator </span></a>
+    <ul class="navbar-nav ml-auto" style="margin-left: 50%;">
+        <li style="margin-left: 30px;">
+            <div class=" bg-white dark:bg-gray-800 sm:rounded-lg">
+                <a href="{!! route('language', ['en']) !!}" style="text-decoration: none; margin-right: 30px;"> {{__('English')}}</a>
+                <a href="{!! route('language', ['vi']) !!}" style="text-decoration: none;"> {{__('Vietnamese')}}</a>
+            </div>
+        </li>
+        <li style="margin-left: 30px;">
+            <a href="">
```
Comment (huyenttm-0901) [inline resources/views/layouts/header.blade.php:13]: viết hết css ra file riêng đi em

File: resources/views/layouts/master.blade.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    </div>
+    <script src="{{ asset('bower_components/bootstrap/dist/js/bootstrap.min.js')}}" type="text/javascript"></script>
+    <script src="{{ asset('bower_components/jquery/dist/jquery.min.js')}}" type="text/javascript"></script>
+    <script src="{{ asset('js/app.js') }}"></script>
+    <script src="{{ asset('bower_components/assets/js/light-bootstrap-dashboard.js') }}"></script>
+    <!-- <script src="{{ asset('bower_components/template_package/assets/js/demo.js') }}"></script> -->
+
+</body>
+
+</html>
```
Comment (huyenttm-0901) [inline resources/views/layouts/master.blade.php:32]: tương tự

**Comment by dvson-sun** [comment]
ready

**Comment by manhtd-1083** [comment]
merged

**Pull request merged**